CHANGELOG.md

```diff
@@ -8,6 +8,22 @@ Try to keep listed changes to a concise bulleted list of simple explanations of
 
 ## [Unreleased]
 
+## [v1.4.1] - 2024-01-04
+
+[Full changelog](https://github.com/openfga/openfga/compare/v1.4.0...v1.4.1)
+
+### Changed
+* Reduce goroutine overhead in ListObjects ([#1173](https://github.com/openfga/openfga/pull/1173))
+
+### Added
+* Support for cancellation/timeouts when evaluating Conditions ([#1237](https://github.com/openfga/openfga/pull/1237))
+* Tracing span info for Condition evaluation ([#1251](https://github.com/openfga/openfga/pull/1251))
+
+### Fixed
```

Comment:
Can you add https://github.com/openfga/openfga/pull/1236?

Reply:
Yep! Missed that one. Thank you